src/indsnetworkmap.jl

```diff
@@ -98,6 +98,10 @@ function dimension_vertices(inm::IndsNetworkMap, dimension::Int)
   return filter(v -> vertex_dimension(inm, v) == dimension, vertices(inm))
```

Comment:
This should now just call the `dimension_vertices` defined below, i.e.
`dimension_vertices(inm::IndsNetworkMap, dimension::Int) = dimension_vertices(inm, [dimension])`

Reply:
This needs to go in the other PR since operator is changed there to avoid merge headaches

Reply:
Was this comment intended for the above? Not sure I understand?

Reply:
I was suggesting we deal with this after the PR from the integrate, but I'll just change this here then